xonsh/prompt/base.py

```diff
@@ -1,13 +1,20 @@
 # -*- coding: utf-8 -*-
-"""Base prompt, provides FORMATTER_DICT"""
+"""Base prompt, provides FORMATTER_DICT and prompt related functions"""
 
+import builtins
+import itertools
 import os
+import re
 import socket
+import string
+import sys
 
 import xonsh.lazyasd as xl
 import xonsh.tools as xt
 import xonsh.platform as xp
 
+from xonsh.tools import format_color, print_exception
+from xonsh.platform import ON_WINDOWS, ON_CYGWIN
```

Comment:
Since `xonsh.tools` and `xonsh.platform` are already imported and renamed above, should probably just use those.